src/Graphics/samples/GraphicsTester.WinUI.Desktop/MainWindow.xaml

```diff
@@ -16,6 +17,7 @@
         </Grid.ColumnDefinitions>
         <ListView x:Name="List" />
         <Canvas Grid.Column="1" Width="5" HorizontalAlignment="Stretch" Background="Black"/>
-        <graphics:W2DGraphicsView x:Name="GraphicsView" Grid.Column="2" />
+        <!--<graphics:W2DGraphicsView x:Name="GraphicsView" Grid.Column="2" />-->
```

Comment:
Can we delete this?

Reply:
We can leave it because both views should work. The type is not replaced, there are now 2.